alma/endpoints/exports.py

```diff
@@ -26,6 +31,14 @@ def create(
 
         if end:
             data["end"] = int(end.timestamp())
+        
+        if custom_fields:
+            if isinstance(custom_fields, str):
+                data["custom_fields"] = custom_fields
+            elif type(custom_fields) in (list, tuple, set):
+                data["custom_fields"] = ",".join(custom_fields)
+            else:
+                raise TypeError(f"Expected comma-separated string or a list/tuple/set of strings for custom_fields, got {type(custom_fields)}")
```

Comment:
(nit) `join` works with an iterable, so what about supporting any iterable for `custom_fields`:

```py
if custom_fields:
    if isinstance(custom_fields, str):
        data["custom_fields"] = custom_fields
    else:
        try:
            data["custom_fields"] = ",".join(custom_fields)
        except TypeError as e:
            raise TypeError(f"Expected comma-separated string or an iterable yielding strings for custom_fields, got {type(custom_fields)}") from e
```

Reply:
Thanks, it's better indeed I'll fix that 👍 

Plus I didn't know about the `raise ... from` syntax! 🙌